GNUmakefile

```diff
@@ -57,4 +60,42 @@ ifeq (,$(wildcard $(GOPATH)/src/$(WEBSITE_REPO)))
 endif
 	@$(MAKE) -C $(GOPATH)/src/$(WEBSITE_REPO) website-provider-test PROVIDER_PATH=$(shell pwd) PROVIDER_NAME=$(PKG_NAME)
 
+docker-build:
+	@docker build -t $(DOCKER_IMAGE_NAME) .
+
+docker-run: docker-show-env check-itm-env
+ifndef ITM_HOST_MODULE_DIR
```

Comment:
this could be moved down to line 99 and it might be a bit cleaner

Reply:
Agreed. Done.